Sources/BraintreePayPalMessaging/BTPayPalMessagingView.swift

```diff
@@ -12,6 +12,8 @@ public class BTPayPalMessagingView: UIView {
 
     // MARK: - Properties
 
+    private var messageView: PayPalMessageView?
```

Comment:
Super nit: can we put this private property under the internal? We generally try to order them public > internal > private